tests/bsim/bluetooth/mesh/tests_scripts/friendship/msg_va_collision.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Copyright 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# Test receives on group and virtual addresses in the LPN
```

Comment:
Could you add a bit more description of the test scenario? It is quite a complex sequence.